<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="0" skipped="2">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-02-23T00:19:32 IST" name="Default suite" finished-at="2021-02-23T00:19:32 IST" duration-ms="158">
    <groups>
    </groups>
    <test started-at="2021-02-23T00:19:32 IST" name="Default test" finished-at="2021-02-23T00:19:32 IST" duration-ms="158">
      <class name="extentReports.GenerateExtentReport">
        <test-method is-config="true" signature="startReport()[pri:0, instance:extentReports.GenerateExtentReport@1d082e88]" started-at="2021-02-23T00:19:32 IST" name="startReport" finished-at="2021-02-23T00:19:32 IST" duration-ms="68" status="FAIL">
          <exception class="java.lang.NoClassDefFoundError">
            <message>
              <![CDATA[io/reactivex/rxjava3/subjects/PublishSubject]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NoClassDefFoundError: io/reactivex/rxjava3/subjects/PublishSubject
at com.aventstack.extentreports.ReactiveSubject.<init>(ReactiveSubject.java:28)
at com.aventstack.extentreports.AbstractProcessor.<init>(AbstractProcessor.java:22)
at com.aventstack.extentreports.ExtentReports.<init>(ExtentReports.java:57)
at extentReports.GenerateExtentReport.startReport(GenerateExtentReport.java:22)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:617)
at org.testng.TestRunner.beforeRun(TestRunner.java:607)
at org.testng.TestRunner.run(TestRunner.java:578)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassNotFoundException: io.reactivex.rxjava3.subjects.PublishSubject
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:602)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
... 28 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NoClassDefFoundError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startReport -->
        <test-method signature="demoReportFail()[pri:0, instance:extentReports.GenerateExtentReport@1d082e88]" started-at="2021-02-23T00:19:32 IST" name="demoReportFail" finished-at="2021-02-23T00:19:32 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.NoClassDefFoundError">
            <message>
              <![CDATA[io/reactivex/rxjava3/subjects/PublishSubject]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NoClassDefFoundError: io/reactivex/rxjava3/subjects/PublishSubject
at com.aventstack.extentreports.ReactiveSubject.<init>(ReactiveSubject.java:28)
at com.aventstack.extentreports.AbstractProcessor.<init>(AbstractProcessor.java:22)
at com.aventstack.extentreports.ExtentReports.<init>(ExtentReports.java:57)
at extentReports.GenerateExtentReport.startReport(GenerateExtentReport.java:22)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:617)
at org.testng.TestRunner.beforeRun(TestRunner.java:607)
at org.testng.TestRunner.run(TestRunner.java:578)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassNotFoundException: io.reactivex.rxjava3.subjects.PublishSubject
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:602)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
... 28 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NoClassDefFoundError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- demoReportFail -->
        <test-method is-config="true" signature="afterEachTest(org.testng.ITestResult)[pri:0, instance:extentReports.GenerateExtentReport@1d082e88]" started-at="2021-02-23T00:19:32 IST" name="afterEachTest" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1614019772570" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterEachTest -->
        <test-method signature="demoReportPass()[pri:0, instance:extentReports.GenerateExtentReport@1d082e88]" started-at="2021-02-23T00:19:32 IST" name="demoReportPass" finished-at="2021-02-23T00:19:32 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.NoClassDefFoundError">
            <message>
              <![CDATA[io/reactivex/rxjava3/subjects/PublishSubject]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NoClassDefFoundError: io/reactivex/rxjava3/subjects/PublishSubject
at com.aventstack.extentreports.ReactiveSubject.<init>(ReactiveSubject.java:28)
at com.aventstack.extentreports.AbstractProcessor.<init>(AbstractProcessor.java:22)
at com.aventstack.extentreports.ExtentReports.<init>(ExtentReports.java:57)
at extentReports.GenerateExtentReport.startReport(GenerateExtentReport.java:22)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:617)
at org.testng.TestRunner.beforeRun(TestRunner.java:607)
at org.testng.TestRunner.run(TestRunner.java:578)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassNotFoundException: io.reactivex.rxjava3.subjects.PublishSubject
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:602)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
... 28 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NoClassDefFoundError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- demoReportPass -->
        <test-method is-config="true" signature="afterEachTest(org.testng.ITestResult)[pri:0, instance:extentReports.GenerateExtentReport@1d082e88]" started-at="2021-02-23T00:19:32 IST" name="afterEachTest" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1614019772582" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterEachTest -->
        <test-method is-config="true" signature="endReport()[pri:0, instance:extentReports.GenerateExtentReport@1d082e88]" started-at="2021-02-23T00:19:32 IST" name="endReport" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1614019772589" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- extentReports.GenerateExtentReport -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
